src/Glpi/Form/ServiceCatalog/ItemRequest.php

```diff
@@ -45,7 +46,8 @@ public function __construct(
         public string $filter = "",
         public ?Category $category = null,
         public int $page = 1,
-        public int $items_per_page = ServiceCatalogManager::ITEMS_PER_PAGE
+        public int $items_per_page = ServiceCatalogManager::ITEMS_PER_PAGE,
+        public ?string $sort_strategy = null
```

Comment:
I think it is preferable than using a `??` operator in the `getSortStrategy()` getter.

```suggestion
        public string $sort_strategy = SortStrategyFactory::DEFAULT_STRATEGY
```